### PR TITLE
[presign] Remove cancelation of orders

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

This PR removes the ability of cancelation of orders for pre-sign workflow.

Before:
image

After:
image

## not included
Onchain cancelation
#733 

# To Test

1. Create an order using pre-sigh (default, even for EOA in this PR)
2. Observe that now is not possible to cancel

# Background
SC wallets can't do offchain signing

### CODE REVIEW

File: src/custom/components/AccountDetails/Transaction/index.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ import {
 } from './styled'
 import { getLimitPrice, getExecutionPrice } from 'state/orders/utils'
 import { DEFAULT_PRECISION } from 'constants/index'
+import { useWalletInfo } from '@src/custom/hooks/useWalletInfo'
```
Comment (alfetopito) [inline src/custom/components/AccountDetails/Transaction/index.tsx:50]: Path?

```suggestion
import { useWalletInfo } from 'hooks/useWalletInfo'
```

**Review by alfetopito — APPROVED** [review]

**Comment by MareenG** [comment]
The layout of the toast notification needs to be adjusted, now it overlaps with the rim. Maybe we can shorten the ID as we do in the Explorer. 
Bildschirmfoto 2021-09-08 um 10 33 16
Bildschirmfoto 2021-09-08 um 10 37 43

**Comment by MareenG** [comment]
Also the pending disappeared 
Bildschirmfoto 2021-09-08 um 10 41 57

**Comment by MareenG** [comment]
> Also the pending disappeared
> Bildschirmfoto 2021-09-08 um 10 41 57

Not always though

**Comment by MareenG** [comment]
The icon looks a bit squeezed maybe it can be made a little bit smaller.
Bildschirmfoto 2021-09-08 um 11 30 43

**Review by MareenG — APPROVED** [review]
The cancelation is successfully removed. I only had some other minor comments :)

**Comment by elena-zh** [comment]
> > Also the pending disappeared
> > Bildschirmfoto 2021-09-08 um 10 41 57
> 
> Not always though

I noticed that it becomes in 'Pending' when status is changed from 'Pre-signing' to 'Open'

**Comment by MareenG** [comment]
> I noticed that it becomes in 'Pending' when status is changed from 'Pre-signing' to 'Open'
True, I haven't closed the sidebar to observe this.

**Comment by anxolin** [comment]
Hei @MareenG all good comments and concerns. Most of them we are aware and are capture one way or another in https://github.com/gnosis/cowswap/issues/1305, some have even been solved in other PRs (like the message thing). The scope of this PR is very small, remove the cancelation button. 

This is an ongoing tasks, and we do it in smaller pieces. The complete functionality will come later. Not sure if you want to test the smaller iterations or just once we consolidate most of the things...

Anyways, feel free to add things here if you feel we forget sth important: https://github.com/gnosis/cowswap/issues/1305

**Comment by MareenG** [comment]
> Not sure if you want to test the smaller iterations or just once we consolidate most of the things...

I think in general it is fine to test smaller parts of a bigger implementation. Maybe just mention the bigger scope in the PR as you did now with #1305 then I better track the whole scope and the issues you are already working on :)

**Pull request merged**